bokehjs/src/lib/models/plots/gmap_plot_canvas.ts

```diff
@@ -4,12 +4,16 @@ import {Context2d} from "core/util/canvas"
 import {GMapPlot} from "./gmap_plot"
 import {PlotView, RangeInfo, FrameBox} from "./plot_canvas"
 
-declare let _bokeh_gmaps_callback: () => void
+declare global {
+  interface Window {
+    _bokeh_gmaps_callback: () => void
+  }
+}
 
 const gmaps_ready = new Signal0({}, "gmaps_ready")
 
 const load_google_api = function(api_key: string): void {
-  _bokeh_gmaps_callback = () => gmaps_ready.emit()
+  window._bokeh_gmaps_callback = () => gmaps_ready.emit()
```

Comment:
In strict mode naked (no `var`, `let`, `const` keyword) variables aren't allowed, resulting in `ReferenceError` exception. It's interesting why it started to fail just now, because this code was changed during transition to TypeScript which was like a year ago. Or maybe it was broken all along, just we didn't test this scenario.